__tests__/Unit/Components/Tasks/TasksContent.test.tsx

```diff
@@ -84,4 +89,60 @@ describe('tasks content', () => {
         );
         expect(task).toBeInTheDocument();
     });
+
+    test('load more button is disabled when there are no more tasks to load', async () => {
+        server.use(noTasksFoundHandler);
+        renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent dev={true} />
+            </Provider>,
+            { query: { dev: 'true' } }
+        );
+        await screen.findByTestId('tabs');
+        const loadMoreButton = screen.getByRole('button', {
+            name: /load more/i,
+        });
+        expect(loadMoreButton).toBeDisabled();
+    });
+
+    test('load more button is enabled when there are more tasks to load', async () => {
+        renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent dev={true} />
+            </Provider>,
+            { query: { dev: 'true' } }
+        );
+        await screen.findByTestId('tabs');
+        const loadMoreButton = screen.getByRole('button', {
+            name: /load more/i,
+        });
+        expect(loadMoreButton).toBeEnabled();
+    });
+
+    test('fetch more tasks when load more button is clicked', async () => {
+        const { findByText } = renderWithRouter(
+            <Provider store={store()}>
+                <TasksContent dev={true} />
+            </Provider>,
+            { query: { dev: 'true' } }
+        );
+        await screen.findByTestId('tabs');
+        const availableButton = screen.getByRole('button', {
+            name: /UNASSINGED/i,
+        });
+        fireEvent.click(availableButton);
+        const task = await findByText(
+            'Design and develop an online booking system'
+        );
+        expect(task).toBeInTheDocument();
+        const loadMoreButton = await screen.getByRole('button', {
+            name: /load more/i,
+        });
+        expect(loadMoreButton).toBeEnabled();
+        fireEvent.click(loadMoreButton);
+        const task2 = await findByText(
+            'Design and develop an online booking system'
+        );
+        expect(task2).toBeInTheDocument();
+    });
```

Comment:
Can we write a test to check if one of the tabs is clicked, then the tasks are only related to that satus?